Tasks/KubernetesV0/src/kubernetescommand.ts

```diff
@@ -58,6 +66,20 @@ function getCommandArguments(): string {
     return tl.getInput("arguments", false);
 }
 
+function isJsonOrYamlOutputFormatSupported(kubecommand) : boolean
+{
+   switch (kubecommand) {
```

Comment:
cannot we do this in task.json?

Reply:
I don't think so. There are no properties by which we can achieve that scenario yet

Reply:
visibility rule?